[WIP] 2020 Enrichment Paper Draft

stompsjo opened the pull request:
Hopefully this PR can serve as a discussion and feedback forum for crafting this paper. I have finished my own review of the draft and hit a roadblock of what to build on next. Certainly there is plenty more work to be done on this paper, but I'm hoping having another reviewer can help guide me in the right direction. I can think of a few to-dos right away:

- [ ] The [theory](https://github.com/stompsjo/ANS.BaM.paper/blob/2020_enrichment_paper/theory.tex) section regurgitates a lot of equations from Raetz and Glaser and I don't want to repeat _too_ much of that content, so I would appreciate some feedback on whether what I have included is sufficient or not.

- [ ] I think there are several sentences that could be supported by references, please let me know if you spot one.

I have updated the results section table and figures to reflect the most up-to-date development on [`mbmore`](https://github.com/stompsjo/mbmore/tree/dU_SWU_correction), and I have updated the corresponding discussion language as well. Thanks for your help!

Some relevant repos:
- All simulations, notebooks, and plots are at this repo and branch: [enirch_inmm:pr5](https://github.com/stompsjo/enrich_inmm/tree/pr5)
- The most up-to-date and stable `mbmore` branch is at [stompsjo/mbmore:dU_SWU_correction](https://github.com/stompsjo/mbmore/tree/dU_SWU_correction)

Finally, if you would like a direct link to the latex compiled PDF, you can download it [here](https://github.com/stompsjo/ANS.BaM.paper/blob/2020_enrichment_paper/paper.pdf)

Commit:
Adding parameter detail to Raetz equation

Commit:
Theory of Z_p

Commit:
Adding r1/r2 information

Commit:
Last inclusion of Raetz information.

Commit:
Pruning theory wording

Commit:
Adding information about ideal stage from ideal cut algorithm

Commit:
Correcting alpha/beta definitions

Commit:
Adding interim definitions to beta equation

Commit:
Adding machines needed per stage information

Commit:
Proofreading misuse models section

Commit:
updating plots for rectifier length design cut

Commit:
formatting abstract and title page

Commit:
reverting italics abstract

Commit:
attempting to resize matrix

Commit:
scaling cascade config schematic figure

Commit:
attempting to resize results table

Commit:
still resizing results table

Commit:
adding a columnwidth req to results table

Commit:
shrinking results table with footnotesize

Commit:
updating numbers in results table

Commit:
updating real cascade numbers in results table

Commit:
incrementing results figure scaling width

Commit:
scaling results figure

Commit:
scaling results figure further

Commit:
scaling third subfigure in results

Commit:
trying one more scaling feature in results table

Commit:
proofreading

bam241 commented:
Quick comment while I quickly go through:
- Fig1: missing x axis label
- Fig2: missing arrows on tails recycling ?
- Fig3: purple not mentioned, (also Tab 4 mentions 5 level (0 to 4) not only 4)

bam241 commented:
also you need to add your name in the author list (as first author?!)

Commit:
updating figure 1 and 3

stompsjo commented:
- ✅  Fig 1: Updated to include axis, it was being cutoff when saving the figure from the python notebook in [enrich_calcs](https://github.com/stompsjo/enrich_calcs/blob/mbmore_unit_tests/alpha_minus_beta.png)
- ❌  Fig 2: @bam241 I don't have this drawing. It looks like a Google Drawing to me but I couldn't find it in the CNERG drive, do you know where it is? I agree that the tails recycling needs arrows here and I will do it if I can find a copy.
- ✅  Fig3: Updated the figure caption. With all the new changes, case B/NR requires a 5th level to reach 90w%, which wasn't the case before. The caption should now accurately reflect what is in Table 4 and the plots.
- ❓  Author name: I added myself as an author alphabetically for now. I don't know whether I've warranted first author or not (yet). I'm afraid to step on anyone's toes 😜

Commit:
adding arrows to flow.png

stompsjo commented:
- ✅  Fig 2: Arrows added

nuclearkatie commented on theory.tex:
```diff
@@ -93,19 +137,20 @@ \subsection{Centrifuges basic properties and definition}
+and external feed, $F_{ext}$:
 
 \begin{equation}
     F_{i} = T_{i+1} + F_{i-1} ~(+ F_{ext})
```
Is `F_{i-1}` meant to be `P_{i-1}`, or am I misreading?

edit: update with the underscore, thanks @bam241 
```suggestion
    F_{i} = T_{i+1} + P_{i-1} ~(+ F_{ext})
```

bam241 replied:
this sounds about right

bam241 replied:
but you missed the `_`

stompsjo replied:
Good question... @bam241 might want to confirm this but I think you're right. The feed of stage `i` should be made up of the product of the upper stage plus the feed of the lower stage. If this was `F_{i-1}` it would ignore the output of that stage (?)

stompsjo replied:
You beat me to it! So,
```suggestion
    F_{i} = T_{i+1} + P_{i-1} ~(+ F_{ext})
```

nuclearkatie committed:
first review pass from abstract up until 2.5.1

stompsjo commented on theory.tex:
```diff
@@ -38,74 +38,114 @@ \subsubsection{Separative power}
+\end{equation}
+ 
+Here the gas constants are molar mass, $M$, temperature $T$, universal gas constant, $R$,
+and the pressure ratio, $x$ (typically $1000:1$) \cite{}. %needs citation?
```
This comes from `glaser.2008`, page 4. But there is no reference there, not sure if he is referencing Raetz or...?
```suggestion
and the pressure ratio, $x$ (typically $1000:1$) \cite{glaser.2008}. %needs citation?
```

stompsjo commented on theory.tex:
```diff
@@ -38,74 +38,114 @@ \subsubsection{Separative power}
-product enrichment factor, $\alpha$, the cut, $\theta$, and the feed rate, $F$:
+It can be shown \cite{avery} that the separative power of a single centrifuge can be
+derived from first principles and expressed as a function of the feed-to-product enrichment factor, $\alpha$, the cut, $\theta$, and the feed rate, $F$:
+%I think we should replicate at least some of that here
```
Good point, although IIRC Avery only states this equation on its own. Will double check in the morning.

stompsjo replied:
Avery/Davies derive this relationship from two equations (page 18): (1) the definition of separative power, which is a function of delta_S (the entropy change) and (2) the entropy. There's some variable changes to express entropy in terms of the cut (theta) and the enrichment factor (alpha) that leads to this equation. I'll try to express these equations in the paper this week (since I have the book).

stompsjo replied:
@nuclearkatie @bam241 I have expanded on the first principles derivation by repeating some of the equations in [Katie's worksheet](https://docs.google.com/document/d/1VoyoLFQV4TStAJnC33IAppMqRXd4VYxUScxVcxpuRGQ/edit?usp=sharing). I should stress that this is _far from a complete derivation_ and skips steps heavily. I think this is a good first step of addressing this but we can discuss further next time we meet.

Commit:
Corrected feed typo.

Co-authored-by: Katie Mummah <radioactivekate@gmail.com>

Commit:
expanding on first principles derivation
